e2e-tests/portal/cypress/e2e/journeys/maker/proceed-to-review/proceed-to-review.spec.js

```diff
@@ -127,22 +176,78 @@ context('Ensure proceed to review button is only visible once facilities are in
   });
 
   it('Ensure proceed to review button is visible', () => {
-    // Login as a `Maker`
-    cy.login(BANK1_MAKER1);
+    cy.loginGoToDealPage(BANK1_MAKER1);
+
+    contract.aboutSupplierDetailsLink().click();
+
+    // Fill in details about supplier
+    contractAboutSupplier.supplierType().select('Exporter');
+    contractAboutSupplier.supplierCompaniesHouseRegistrationNumber().clear().type('04301762', { delay: 0 });
+    contractAboutSupplier.supplierSearchCompaniesHouse().click();
+    contractAboutSupplier.supplierAddress().country().select('United Kingdom');
+    contractAboutSupplier.supplierCorrespondenceAddressSame().click();
+    contractAboutSupplier.smeTypeSmall().click();
+    contractAboutSupplier.supplyContractDescription().type('Supply Contract Description', { delay: 0 });
```

Comment:
`cy.keyboardInput` instead of `.type` please, for all instances

Reply:
Replaced with `cy.keyboardInput`